presto-mysql/src/main/java/com/facebook/presto/plugin/mysql/MySqlClient.java

```diff
@@ -142,6 +155,63 @@ protected SchemaTableName getSchemaTableName(ResultSet resultSet)
                 resultSet.getString("TABLE_NAME").toLowerCase(ENGLISH));
     }
 
+    @Override
+    protected JdbcTypeHandle toJdbcTypeHandle(ConnectorSession session, Type type)
```

Comment:
This method would be much simpler if it was limited to the functionality that is required now. I.e. if it returned `WriteFunction` directly, instead of `JdbcTypeHandle`.
Thoughts? What should be the name? `getWriteFunction`? Should the name convey this is used for CTAS only?